docs/ARCHIVING.md

```diff
@@ -60,7 +60,7 @@ NOTE: At the time of writing, this is a completely manual process. This is becau
 
 - In `docs/` sub-directory:
   - Replace `/img` paths with `/0.1/img` paths
-- Run `npm run build:withoutAuth` and store the `build/` directory as `site/0.1`
+- Run `yarn run build:withoutAuth` and store the `build/` directory as `site/0.1`
```

Comment:
@jedevc I don't think the changes in this file are correct. These archives are built from specific past releases, presumably `yarn` will not exist at those time points?

Reply:
Woops, fixed :tada: